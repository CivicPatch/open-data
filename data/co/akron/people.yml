---
- name: Brandon Hill
  roles:
  - Mayor
  divisions:
  image:
  cdn_image:
  email:
  phone_number:
  website:
  start_date:
  end_date: '2026'
  sources:
  - https://townofakron.colorado.gov/government/board-of-trustees
  updated_at: '2025-07-09T00:16:44Z'
- name: Jared Jefferson
  roles:
  - Mayor Pro Tempore
  - Trustee
  divisions:
  image:
  cdn_image:
  email:
  phone_number:
  website:
  start_date:
  end_date: '2028'
  sources:
  - https://townofakron.colorado.gov/government/board-of-trustees
  updated_at: '2025-07-09T00:16:44Z'
- name: Ariella Gonzales-Vondy
  roles:
  - Trustee
  divisions:
  image:
  cdn_image:
  email:
  phone_number:
  website:
  start_date:
  end_date: '2026'
  sources:
  - https://townofakron.colorado.gov/government/board-of-trustees
  updated_at: '2025-07-09T00:16:44Z'
- name: Braden Brent
  roles:
  - Trustee
  divisions:
  image:
  cdn_image:
  email:
  phone_number:
  website:
  start_date:
  end_date: '2028'
  sources:
  - https://townofakron.colorado.gov/government/board-of-trustees
  updated_at: '2025-07-09T00:16:44Z'
- name: Crystann Benson
  roles:
  - Trustee
  divisions:
  image:
  cdn_image:
  email:
  phone_number:
  website:
  start_date:
  end_date: '2028'
  sources:
  - https://townofakron.colorado.gov/government/board-of-trustees
  updated_at: '2025-07-09T00:16:44Z'
- name: David Kembel
  roles:
  - Trustee
  divisions:
  image:
  cdn_image:
  email:
  phone_number:
  website:
  start_date:
  end_date: '2026'
  sources:
  - https://townofakron.colorado.gov/government/board-of-trustees
  updated_at: '2025-07-09T00:16:44Z'
- name: Jennifer Hansen
  roles:
  - Trustee
  divisions:
  image:
  cdn_image:
  email:
  phone_number:
  website:
  start_date:
  end_date: '2026'
  sources:
  - https://townofakron.colorado.gov/government/board-of-trustees
  updated_at: '2025-07-09T00:16:44Z'
