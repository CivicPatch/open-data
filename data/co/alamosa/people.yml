---
- name: Ty Coleman
  roles:
  - Mayor
  divisions:
  image: https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_0479-683x1024.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/alamosa/images/330c748392a7e8051c8e5e551c43e7669988e7bfa6ade0f81733b5695e6fff86.jpg
  email:
  phone_number: "(719) 313-7241"
  website: https://cityofalamosa.org/mayor
  start_date:
  end_date:
  sources:
  - https://cityofalamosa.org/team/city-council
  - https://cityofalamosa.org/mayor
  updated_at: '2025-06-30T17:01:59Z'
- name: Jan Vigil
  roles:
  - Council Member
  divisions:
  - At-large
  image: https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_0490-683x1024.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/alamosa/images/2927d180be796f0b55a7442a10c4b4a3f86a27e297fdf35cd2364ebe218e0179.jpg
  email:
  phone_number: "(970) 946-5611"
  website: https://cityofalamosa.org/councilor-at-large-vigil
  start_date:
  end_date:
  sources:
  - https://cityofalamosa.org/team/city-council
  - https://cityofalamosa.org/councilor-at-large-vigil
  updated_at: '2025-06-30T17:01:59Z'
- name: Michael Carson
  roles:
  - Council Member
  divisions:
  - At-large
  image: https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_9183.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/alamosa/images/3e64490dc1cf5225a9c48fe3dac710d83b31ca66bb932b477d871d307f32fd79.jpg
  email:
  phone_number: "(719) 992-4839"
  website: https://cityofalamosa.org/councilor-at-large-daniel
  start_date:
  end_date:
  sources:
  - https://cityofalamosa.org/team/city-council
  - https://cityofalamosa.org/councilor-at-large-daniel
  updated_at: '2025-06-30T17:01:59Z'
- name: Liz Hensley
  roles:
  - Council Member
  divisions:
  - Ward 1
  image: https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_0486.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/alamosa/images/cf42cdae3deb9921575019fe58a4f31f51cdd37351bffe1a27022af08e1aa675.jpg
  email:
  phone_number: "(719) 580-0201"
  website: https://cityofalamosa.org/city-councilor-ward-1
  start_date:
  end_date:
  sources:
  - https://cityofalamosa.org/team/city-council
  - https://cityofalamosa.org/city-councilor-ward-1
  updated_at: '2025-06-30T17:01:59Z'
- name: Dawn Krebs
  roles:
  - Council Member
  divisions:
  - Ward 2
  image: https://cityofalamosa.org/wp-content/uploads/2022/02/IMG_9105-683x1024.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/alamosa/images/7ba138d6964067a1250396d3b0856287a0967787572036cb47f6703b9a72b215.jpg
  email:
  phone_number: "(719) 580-2194"
  website: https://cityofalamosa.org/city-councilor-ward-2
  start_date:
  end_date:
  sources:
  - https://cityofalamosa.org/team/city-council
  - https://cityofalamosa.org/city-councilor-ward-2
  updated_at: '2025-06-30T17:01:59Z'
- name: Jackie Vigil
  roles:
  - Council Member
  divisions:
  - Ward 3
  image: https://cityofalamosa.org/wp-content/uploads/2022/02/JackieVigil-683x1024.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/alamosa/images/ddc9baa5a1dfcc7276d8f26e050aa867575cca21be9ad560fd044f199fbd1812.jpg
  email:
  phone_number: "(719) 992-8008"
  website: https://cityofalamosa.org/mayorprotem-ward3
  start_date:
  end_date:
  sources:
  - https://cityofalamosa.org/mayorprotem-ward3
  - https://cityofalamosa.org/team/city-council
  - https://cityofalamosa.org/mayor
  updated_at: '2025-06-30T17:01:59Z'
- name: Jamie Dominguez
  roles:
  - Council Member
  divisions:
  - Ward 4
  image: https://cityofalamosa.org/wp-content/uploads/2022/02/JamieDominguez-scaled.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/alamosa/images/986fba6ff59df47feba3453158ab68d03f8eab5a50e105fd1667a724dd7a46c7.jpg
  email:
  phone_number: "(719) 937-3039"
  website: https://cityofalamosa.org/city-councilor-ward-4
  start_date:
  end_date:
  sources:
  - https://cityofalamosa.org/team/city-council
  - https://cityofalamosa.org/city-councilor-ward-2
  - https://cityofalamosa.org/city-councilor-ward-4
  updated_at: '2025-06-30T17:01:59Z'
