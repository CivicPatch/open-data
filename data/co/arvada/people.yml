---
- name: Lauren Simpson
  roles:
  - Mayor
  divisions:
  image: https://www.arvada.org/ImageRepository/Document?documentId=4946
  cdn_image: https://cdn.civicpatch.org/open-data/co/arvada/images/f564839334847418a0d26dd30a7977e6f5d0d1f5999778d285542ff45b1787dd.jpg
  email: Council@arvada.org
  phone_number:
  website: https://www.arvada.org/158/About-City-Council
  start_date:
  end_date:
  sources:
  - https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-
  - https://www.arvada.org/158/About-City-Council
  updated_at: '2025-06-27T20:13:15Z'
- name: Randy Moorman
  roles:
  - Mayor Pro Tempore
  divisions:
  - District 1
  image: https://www.arvada.org/ImageRepository/Document?documentId=4946
  cdn_image: https://cdn.civicpatch.org/open-data/co/arvada/images/f564839334847418a0d26dd30a7977e6f5d0d1f5999778d285542ff45b1787dd.jpg
  email: Council@arvada.org
  phone_number:
  website: https://www.arvada.org/158/About-City-Council
  start_date:
  end_date:
  sources:
  - https://www.arvada.org/158/About-City-Council
  updated_at: '2025-06-27T20:13:15Z'
- name: Brad Rupert
  roles:
  - Council Member
  divisions:
  - At-large
  image: https://www.arvada.org/ImageRepository/Document?documentId=4946
  cdn_image: https://cdn.civicpatch.org/open-data/co/arvada/images/f564839334847418a0d26dd30a7977e6f5d0d1f5999778d285542ff45b1787dd.jpg
  email: Council@arvada.org
  phone_number:
  website: https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-
  start_date: '2024-12-16'
  end_date: 2025-11
  sources:
  - https://www.arvada.org/1174/Brad-Rupert-Appointed-to-Fill-Vacant-At-
  - https://www.arvada.org/158/About-City-Council
  updated_at: '2025-06-27T20:13:15Z'
- name: Sharon Davis
  roles:
  - Council Member
  divisions:
  - At-large
  image: https://www.arvada.org/ImageRepository/Document?documentId=4946
  cdn_image: https://cdn.civicpatch.org/open-data/co/arvada/images/f564839334847418a0d26dd30a7977e6f5d0d1f5999778d285542ff45b1787dd.jpg
  email: Council@arvada.org
  phone_number:
  website: https://www.arvada.org/158/About-City-Council
  start_date:
  end_date:
  sources:
  - https://www.arvada.org/158/About-City-Council
  updated_at: '2025-06-27T20:13:15Z'
- name: Shawna Ambrose
  roles:
  - Council Member
  divisions:
  - District 2
  image: https://www.arvada.org/ImageRepository/Document?documentId=4946
  cdn_image: https://cdn.civicpatch.org/open-data/co/arvada/images/f564839334847418a0d26dd30a7977e6f5d0d1f5999778d285542ff45b1787dd.jpg
  email: Council@arvada.org
  phone_number:
  website: https://www.arvada.org/158/About-City-Council
  start_date:
  end_date:
  sources:
  - https://www.arvada.org/158/About-City-Council
  updated_at: '2025-06-27T20:13:15Z'
- name: John Marriott
  roles:
  - Council Member
  divisions:
  - District 3
  image: https://www.arvada.org/ImageRepository/Document?documentId=4946
  cdn_image: https://cdn.civicpatch.org/open-data/co/arvada/images/f564839334847418a0d26dd30a7977e6f5d0d1f5999778d285542ff45b1787dd.jpg
  email: Council@arvada.org
  phone_number:
  website: https://www.arvada.org/158/About-City-Council
  start_date:
  end_date:
  sources:
  - https://www.arvada.org/158/About-City-Council
  updated_at: '2025-06-27T20:13:15Z'
- name: Bob Fifer
  roles:
  - Council Member
  divisions:
  - District 4
  image: https://www.arvada.org/ImageRepository/Document?documentId=4946
  cdn_image: https://cdn.civicpatch.org/open-data/co/arvada/images/f564839334847418a0d26dd30a7977e6f5d0d1f5999778d285542ff45b1787dd.jpg
  email: Council@arvada.org
  phone_number:
  website: https://www.arvada.org/158/About-City-Council
  start_date:
  end_date:
  sources:
  - https://www.arvada.org/158/About-City-Council
  updated_at: '2025-06-27T20:13:15Z'
