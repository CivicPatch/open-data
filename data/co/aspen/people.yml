---
- name: Rachael E. Richards
  roles:
  - Mayor
  divisions:
  image: https://aspen.gov/ImageRepository/Document?documentId=15031
  cdn_image: https://cdn.civicpatch.org/open-data/co/aspen/images/f78a0dd46a92f19a4f8e92dddafe71a13b49afb72a644ed879718176668db642.jpg
  email: council@aspen.gov
  phone_number: "(970) 471-8329"
  website: https://aspen.gov/674/City-Council
  start_date: 2025-04
  end_date: 2027-04
  sources:
  - https://aspen.gov/674/City-Council
  updated_at: '2025-06-30T18:41:58Z'
- name: John Doyle
  roles:
  - Mayor Pro Tempore
  divisions:
  image: https://aspen.gov/ImageRepository/Document?documentId=15031
  cdn_image: https://cdn.civicpatch.org/open-data/co/aspen/images/f78a0dd46a92f19a4f8e92dddafe71a13b49afb72a644ed879718176668db642.jpg
  email: council@aspen.gov
  phone_number: "(970) 948-6251"
  website: https://aspen.gov/674/City-Council
  start_date: 2025-04
  end_date: 2029-04
  sources:
  - https://aspen.gov/674/City-Council
  updated_at: '2025-06-30T18:41:58Z'
- name: Bill Guth
  roles:
  - Council Member
  divisions:
  image: https://aspen.gov/ImageRepository/Document?documentId=15031
  cdn_image: https://cdn.civicpatch.org/open-data/co/aspen/images/f78a0dd46a92f19a4f8e92dddafe71a13b49afb72a644ed879718176668db642.jpg
  email: council@aspen.gov
  phone_number: "(970) 300-2120"
  website: https://aspen.gov/674/City-Council
  start_date: 2023-04
  end_date: 2027-04
  sources:
  - https://aspen.gov/674/City-Council
  updated_at: '2025-06-30T18:41:58Z'
- name: Christine Benedetti
  roles:
  - Council Member
  divisions:
  image: https://aspen.gov/ImageRepository/Document?documentId=15031
  cdn_image: https://cdn.civicpatch.org/open-data/co/aspen/images/f78a0dd46a92f19a4f8e92dddafe71a13b49afb72a644ed879718176668db642.jpg
  email: council@aspen.gov
  phone_number: "(206) 349-5212"
  website: https://aspen.gov/674/City-Council
  start_date: 2025-04
  end_date: 2029-04
  sources:
  - https://aspen.gov/674/City-Council
  updated_at: '2025-06-30T18:41:58Z'
- name: Sam Rose
  roles:
  - Council Member
  divisions:
  image: https://aspen.gov/ImageRepository/Document?documentId=15031
  cdn_image: https://cdn.civicpatch.org/open-data/co/aspen/images/f78a0dd46a92f19a4f8e92dddafe71a13b49afb72a644ed879718176668db642.jpg
  email: council@aspen.gov
  phone_number: "(802) 752-7026"
  website: https://aspen.gov/674/City-Council
  start_date: 2023-04
  end_date: 2027-04
  sources:
  - https://aspen.gov/674/City-Council
  updated_at: '2025-06-30T18:41:58Z'
