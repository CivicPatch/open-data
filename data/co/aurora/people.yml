---
- name: Mike Coffman
  roles:
  - Mayor
  divisions:
  image: https://cdnsm5-hosted.civiclive.com/UserFiles/Servers/Server_1881137/Image/City%20Hall/Mayor%20&%20City%20Council/Mayor%20and%20Council%20Member/Mike-Coffman.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/aurora/images/6abc8513e059194c487bdfad72d7bbbb70b32060e00abc5785f07261b443d840.jpg
  email: mcoffman@auroragov.org
  phone_number: "(303) 739-7015"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448210
  start_date: '2023'
  end_date: 2027-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/mike_coffman
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448210
  updated_at: '2025-06-27T19:42:58Z'
- name: Steve Sundberg
  roles:
  - Mayor Pro Tempore
  - Council Member
  divisions:
  - Ward 2
  image:
  cdn_image:
  email: Steve.Sundberg@auroragov.org
  phone_number: "(303) 525-6659"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174017
  start_date:
  end_date: 2025-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/steve_sundberg
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174017
  updated_at: '2025-06-27T19:42:58Z'
- name: Alison Coombs
  roles:
  - Council Member
  divisions:
  - At-large
  image: https://cdnsm5-hosted.civiclive.com/UserFiles/Servers/Server_1881137/Templates/images/City-of-Aurora-Logo.png
  cdn_image: https://cdn.civicpatch.org/open-data/co/aurora/images/55c96044e887ddd473062b142f085bc1a121e71ae6ad1eded19201eebb6fde21.png
  email: acoombs@auroragov.org
  phone_number: "(720) 505-3332"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448307
  start_date:
  end_date: 2027-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/alison_coombs
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448307
  updated_at: '2025-06-27T19:42:58Z'
- name: Amsalu Kassaw
  roles:
  - Council Member
  divisions:
  - At-large
  image:
  cdn_image:
  email: akassaw@auroragov.org
  phone_number: "(303) 739-7015"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=20651707
  start_date:
  end_date: 2025-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/amsalu_kassaw
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=20651707
  updated_at: '2025-06-27T19:42:58Z'
- name: Curtis Gardner
  roles:
  - Council Member
  divisions:
  - At-large
  image:
  cdn_image:
  email: cgardner@auroragov.org
  phone_number: "(303) 739-7015"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448377
  start_date:
  end_date: 2027-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/curtis_gardner
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448377
  updated_at: '2025-06-27T19:42:58Z'
- name: Danielle Jurinsky
  roles:
  - Council Member
  divisions:
  - At-large
  image:
  cdn_image:
  email: djurinsk@auroragov.org
  phone_number: "(303) 739-7015"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174231
  start_date:
  end_date: 2025-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/danielle_jurinsky
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174231
  updated_at: '2025-06-27T19:42:58Z'
- name: Crystal Murillo
  roles:
  - Council Member
  divisions:
  - Ward 1
  image:
  cdn_image:
  email: cmurillo@auroragov.org
  phone_number: "(303) 739-7502"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448231
  start_date:
  end_date: 2025-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/crystal_murillo
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448231
  updated_at: '2025-06-27T19:42:58Z'
- name: Ruben Medina
  roles:
  - Council Member
  divisions:
  - Ward 3
  image:
  cdn_image:
  email: rlmedina@auroragov.org
  phone_number: "(303) 739-7015"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174140
  start_date:
  end_date: 2025-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/ruben_medina
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=18174140
  updated_at: '2025-06-27T19:42:58Z'
- name: Stephanie Hancock
  roles:
  - Council Member
  divisions:
  - Ward 4
  image: https://cdnsm5-hosted.civiclive.com/UserFiles/Servers/Server_1881137/Image/City%20Hall/Mayor%20&%20City%20Council/Mayor%20and%20Council%20Member/2024/Stephanie-Hancock_2024.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/aurora/images/54059d65ae8281541957bec88110a5a23404e64532b7f4ed08439f32a2cf319e.png
  email: shancock@auroragov.org
  phone_number: "(303) 739-7015"
  website: https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/stephanie_hancock
  start_date:
  end_date: 2027-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/stephanie_hancock
  updated_at: '2025-06-27T19:42:58Z'
- name: Angela Lawson
  roles:
  - Council Member
  divisions:
  - Ward 5
  image:
  cdn_image:
  email: alawson@auroragov.org
  phone_number: "(303) 739-7518"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448434
  start_date:
  end_date: 2027-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/angela_lawson
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448434
  updated_at: '2025-06-27T19:42:58Z'
- name: Fran√ßoise Bergan
  roles:
  - Council Member
  divisions:
  - Ward 6
  image:
  cdn_image:
  email: fbergan@auroragov.org
  phone_number: "(303) 739-7015"
  website: https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448317
  start_date:
  end_date: 2027-11
  sources:
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members
  - https://www.auroragov.org/city_hall/mayor___city_council/mayor___council_members/francoise_bergan
  - https://www.auroragov.org/cms/One.aspx?portalId=16242704&pageId=16448317
  updated_at: '2025-06-27T19:42:58Z'
