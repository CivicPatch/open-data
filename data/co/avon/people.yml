---
- name: Tamra Underwood
  roles:
  - Mayor
  divisions:
  image: https://www.avon.org/ImageRepository/Document?documentId=22948
  cdn_image: https://cdn.civicpatch.org/open-data/co/avon/images/39ba58f223f6f18378895992f61be5789375f0651953196794e80b5a1cf4157b.jpg
  email: tunderwood@avon.org
  phone_number:
  website: https://www.avon.org/directory.aspx?EID=163
  start_date: '2022'
  end_date: '2026'
  sources:
  - https://www.avon.org/141/Avon-Town-Council
  - https://www.avon.org/directory.aspx?EID=163
  updated_at: '2025-06-30T17:33:54Z'
- name: Rich Carroll
  roles:
  - Mayor Pro Tempore
  divisions:
  image: https://www.avon.org/ImageRepository/Document?documentID=24984
  cdn_image: https://cdn.civicpatch.org/open-data/co/avon/images/bdfb21c2b7ec7c9bc38c8e29bafbd1c738affd079f77a700b09224fce5904cab.jpg
  email: rcarroll@avon.org
  phone_number:
  website: https://www.avon.org/directory.aspx?EID=7
  start_date: '2022'
  end_date: '2026'
  sources:
  - https://www.avon.org/141/Avon-Town-Council
  - https://www.avon.org/directory.aspx?EID=7
  updated_at: '2025-06-30T17:33:54Z'
- name: Chico Thuon
  roles:
  - Council Member
  divisions:
  image: https://www.avon.org/ImageRepository/Document?documentID=19238
  cdn_image: https://cdn.civicpatch.org/open-data/co/avon/images/45247af7176e74b152dc0a9c36b6ba668f9929162752aaeb9cdcdac75f7a523a.jpg
  email: cthuon@avon.org
  phone_number:
  website:
  start_date: '2022'
  end_date: '2026'
  sources:
  - https://www.avon.org/141/Avon-Town-Council
  updated_at: '2025-06-30T17:33:54Z'
- name: Gary Brooks
  roles:
  - Council Member
  divisions:
  image: https://www.avon.org/ImageRepository/Document?documentId=24748
  cdn_image: https://cdn.civicpatch.org/open-data/co/avon/images/39a897470471be14799a02f4f81060ed16506e5b6673cd88e71b6405ac4e686c.jpg
  email: gbrooks@avon.org
  phone_number:
  website: https://www.avon.org/directory.aspx?EID=373
  start_date: '2024'
  end_date: '2028'
  sources:
  - https://www.avon.org/141/Avon-Town-Council
  - https://www.avon.org/directory.aspx?EID=373
  updated_at: '2025-06-30T17:33:54Z'
- name: Kevin Hyatt
  roles:
  - Council Member
  divisions:
  image: https://www.avon.org/ImageRepository/Document?documentId=24747
  cdn_image: https://cdn.civicpatch.org/open-data/co/avon/images/4efb75d6e701f38363a43c89a7f1b5551a035c320333b6665ad64c7fd840f712.jpg
  email: khyatt@avon.org
  phone_number:
  website:
  start_date: '2024'
  end_date: '2028'
  sources:
  - https://www.avon.org/141/Avon-Town-Council
  updated_at: '2025-06-30T17:33:54Z'
- name: Lindsay Hardy
  roles:
  - Council Member
  divisions:
  image: https://www.avon.org/ImageRepository/Document?documentId=24973
  cdn_image: https://cdn.civicpatch.org/open-data/co/avon/images/01ceb80abdd27838c2c393155364d665967a1cdf69342afec9357721ebea78c9.jpg
  email: lhardy@avon.org
  phone_number:
  website: https://www.avon.org/directory.aspx?EID=204
  start_date: '2024'
  end_date: '2028'
  sources:
  - https://www.avon.org/141/Avon-Town-Council
  - https://www.avon.org/directory.aspx?EID=204
  updated_at: '2025-06-30T17:33:54Z'
- name: Ruth Stanley
  roles:
  - Council Member
  divisions:
  image: https://www.avon.org/ImageRepository/Document?documentId=22947
  cdn_image: https://cdn.civicpatch.org/open-data/co/avon/images/35fb0f6cf6cad4833c6587a396af9445d8a05d61924b98bfcb554755434fd902.jpg
  email: rstanley@avon.org
  phone_number:
  website: https://www.avon.org/directory.aspx?EID=339
  start_date: '2022'
  end_date: '2026'
  sources:
  - https://www.avon.org/141/Avon-Town-Council
  - https://www.avon.org/directory.aspx?EID=339
  updated_at: '2025-06-30T17:33:54Z'
