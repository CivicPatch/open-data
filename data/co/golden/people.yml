---
- name: Laura Weinberg
  roles:
  - Mayor
  divisions:
  image: https://www.cityofgolden.gov/set_site_up/government/2024_COG_LauraWeinberg_jmp_001_sm%20-%20Copy.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/golden/images/524b726d1ba24fa2c58b46e36e87cb10a5df0c639b0ff8ce98e8745800122faf.jpg
  email:
  phone_number: "(303) 384-8000"
  website: https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  start_date:
  end_date:
  sources:
  - https://www.cityofgolden.net
  updated_at: '2025-06-28T18:44:24Z'
- name: Robert Reed
  roles:
  - Mayor Pro Tempore
  - Council Member
  divisions:
  - Ward 1
  image: https://www.cityofgolden.gov/Images/Government/City%20leadership/Reed-2022-400x533%20-%20Copy.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/golden/images/71c41aaa1c7ea2ecd4e0ba357863db894798b759c404ed2b33ab52f1f93bd61e.jpg
  email:
  phone_number: "(303) 384-8000"
  website: https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  start_date:
  end_date:
  sources:
  - https://www.cityofgolden.net
  updated_at: '2025-06-28T18:44:24Z'
- name: Lisa Vitry
  roles:
  - Council Member
  divisions:
  - District 1
  image: https://www.cityofgolden.gov/Images/Government/Document%20Center/2024_COG_LisaVitry_jmp_002_sm%20-%20web%20-%20Copy.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/golden/images/246fa822570e854a919434bd63563ab1f7241869ca584237b38b3ff12ddb5db0.jpg
  email:
  phone_number: "(303) 384-8000"
  website: https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  start_date:
  end_date:
  sources:
  - https://www.cityofgolden.net
  updated_at: '2025-06-28T18:44:24Z'
- name: Patty Evans
  roles:
  - Council Member
  divisions:
  - District 2
  image: https://www.cityofgolden.gov/Images/Government/Document%20Center/2024_COG_PattyEvans_jmp_002_sm_websquare.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/golden/images/b4747e833ea51de530c1a40778035e39a68e4bf70712108d9392acfa36fb9fef.jpg
  email:
  phone_number: "(303) 384-8000"
  website: https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  start_date:
  end_date:
  sources:
  - https://www.cityofgolden.net
  updated_at: '2025-06-28T18:44:24Z'
- name: Paul Haseman
  roles:
  - Council Member
  divisions:
  - Ward 2
  image: https://www.cityofgolden.gov/Images/Government/City%20leadership/Haseman-2022-400x533%20-%20Copy.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/golden/images/e48564435523ae664bfbd32aedae3a4e5dfb650f7ef9376df45c251211d77808.jpg
  email:
  phone_number: "(303) 384-8000"
  website: https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  start_date:
  end_date:
  sources:
  - https://www.cityofgolden.net
  updated_at: '2025-06-28T18:44:24Z'
- name: Don Cameron
  roles:
  - Council Member
  divisions:
  - Ward 3
  image: https://www.cityofgolden.gov/Images/Government/City%20leadership/2024_COG_DonCameron_jmp_002_sm-websquare.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/golden/images/a5af0fe789c9a22abf2b3ee4b94217433cf8744feace546e0ce4561c860400ae.jpg
  email:
  phone_number: "(303) 384-8000"
  website: https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  start_date:
  end_date:
  sources:
  - https://www.cityofgolden.net
  updated_at: '2025-06-28T18:44:24Z'
- name: Bill Fisher
  roles:
  - Council Member
  divisions:
  - Ward 4
  image: https://www.cityofgolden.gov/Images/Government/City%20leadership/2024_COG_BillFisher_jmp_002_sm.jpg
  cdn_image: https://cdn.civicpatch.org/open-data/co/golden/images/49c23fb655ceaa34bef5b041c5acc1349c9983dfca35aff12f8dfbaf157a8d4d.jpg
  email: BFisher@cityofgolden.net
  phone_number: "(303) 384-8000"
  website: https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  start_date:
  end_date:
  sources:
  - https://www.cityofgolden.net
  - https://www.cityofgolden.gov/government/city_leadership/council_commentaries.php
  updated_at: '2025-06-28T18:44:24Z'
